signhost.php

```diff
@@ -345,11 +345,11 @@ function __construct($type) {
 	}
 }
 
-class iDEAL extends Verification implements JsonSerializable {
+class IDealVerification extends Verification implements JsonSerializable {
```

Comment:
Doesn't changing the classname cause backwards compatibility issues?
Or do we accept this as long it isn't a package on composer yet?